plugins/org.locationtech.udig.tool.edit/src/org/locationtech/udig/tools/edit/BehaviourCommand.java

```diff
@@ -12,73 +13,76 @@
 import java.util.LinkedList;
 import java.util.List;
 
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.core.runtime.SubMonitor;
 import org.locationtech.udig.project.command.AbstractCommand;
 import org.locationtech.udig.project.command.PostDeterminedEffectCommand;
 import org.locationtech.udig.project.command.UndoableMapCommand;
 import org.locationtech.udig.tool.edit.internal.Messages;
 
-import org.eclipse.core.runtime.IProgressMonitor;
-import org.eclipse.core.runtime.SubProgressMonitor;
-
 /**
- * Command used by (@link org.locationtech.udig.tools.edit.EditToolHandler} and 
- * {@link org.locationtech.udig.tools.edit.OrderedCompositeEventBehavior} for executing the 
- * behaviours in a tool.
- * 
+ * Command used by (@link org.locationtech.udig.tools.edit.EditToolHandler} and
+ * {@link org.locationtech.udig.tools.edit.OrderedCompositeEventBehavior} for executing the
+ * behaviors in a tool.
```

Comment:
I'd avoid changing comments from british to american english while keeping the class name as it is. Since we don't have any convention for naming classes except that they should be English lets keep it as it is and of cause consitent ;)